src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9084,6 +9084,11 @@ public enum MethodImplOptions
         AggressiveOptimization = 512,
         InternalCall = 4096,
     }
+    [System.AttributeUsage(System.AttributeTargets.Method, Inherited=false)]
+    public sealed class ModuleInitializerAttribute : Attribute
```

Comment:
```suggestion
    public sealed class ModuleInitializerAttribute : System.Attribute
```
Otherwise, this would show up as diff next time somebody tries to auto-regenerate this file.